examples/adb-overwatch/README.md

```diff
@@ -0,0 +1,29 @@
+# Deploying Overwatch on Azure Databricks
+
+This example contains Terraform code used to deploy Overwatch using the [adb-overwatch module](../../modules/adb-overwatch)
+
+
+## Module content
+
+This code uses the [multi-workspace deployment of Overwatch](https://databrickslabs.github.io/overwatch/deployoverwatch/cloudinfra/azure/#reference-architecturehttps://databrickslabs.github.io/overwatch/deployoverwatch/cloudinfra/azure/#reference-architecture). Overwatch runs in a dedicated Azure Databricks workspace, and monitors the specified workspaces in the config file `modules/adb-overwatch/config/overwatch_deployment_config.csv`.
+
+  ![Overwatch_Arch_Azure](https://user-images.githubusercontent.com/103026825/230571464-5892c5c7-82c2-4808-9003-61b501b75f69.png?raw=true)
```

Comment:
@GeekSheikh My suggestion was about moving the image only. It makes sense to leave this module in this repo because we're publishing it to the Terraform registry from which people can find & reuse it easier.  When it's published, we'll add the links from Overwatch docs & repo to it